packages/amplify-e2e-tests/src/__tests__/__snapshots__/resolvers.test.ts.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`user created resolvers overriding generated resolvers overriding a resolver should not create duplicate function 1`] = `
```

Comment:
for small snapshots, I prefer inline snapshots as it makes the tests more readable / quicker to digest